vault/resource_kv_secret_v2_test.go

```diff
@@ -16,6 +16,9 @@ func TestAccKVSecretV2(t *testing.T) {
 	mount := acctest.RandomWithPrefix("tf-kvv2")
 	name := acctest.RandomWithPrefix("tf-secret")
 
+	updatedMount := acctest.RandomWithPrefix("tf-cloud-metadata")
+	updatedName := acctest.RandomWithPrefix("tf-database-creds")
+
```

Comment:
Should we also be asserting something about `metadata` in the first step below?, or perhaps in all `TestStep`s?

Reply:
Sorry for the delay on this! Added some checks to test `metadata` in the test steps